src/test/kotlin/org/jetbrains/research/transformations/util/ParserSetup.kt

```diff
@@ -77,13 +77,13 @@ object ParserSetup {
      * Checks if parser file is in the target place and it is executable.
      * if not - makes it so.
      */
-    fun checkSetup() {
+    fun checkSetup(toUpdateRepository: Boolean = false) {
         logger.info("Checking correctness of a parser setup")
-        unzipParserRepo()
+        unzipParserRepo(toUpdateRepository)
```

Comment:
Sorry, but I don't get why we should unzip pythonparser repo every time we run tests? Shouldn't we just check that there is a pythonparser file in the tmp?